ql/src/test/queries/clientpositive/timestamp_julian.q

```diff
@@ -0,0 +1,22 @@
+--! qt:timezone:Asia/Singapore
+
+-- Test 1
+-- insert into default.test_sgt700 values ('0200-03-01 00:00:00')
+CREATE EXTERNAL TABLE `TEST_SGT`(`currtime` timestamp) ROW FORMAT SERDE 'org.apache.hadoop.hive.ql.io.parquet.serde.ParquetHiveSerDe' STORED AS
```

Comment:
instead of q test, do it in TestParquetTimestampsHive2Compatibility

Reply:
Done, I have added unit test that covers testing 1st march across 3000 years . 

Also, can we please keep both q test and unit test ?
The datafiles for q tests are only 400 bytes each.

Reply:
we already have some `parquet_historical_timestamp_legacy.parq`, maybe you can set some meaningful names in new files and add your test cases to `parquet_historical_timestamp.q`

Reply:
Done